ui/admin/tests/acceptance/accounts/selectors.js

```diff
@@ -6,10 +6,12 @@
 export const APP_HEADER_DROPDOWN =
   '.rose-header-utilities details:first-child summary';
 
-export const MANAGE_DROPDOWN =
+export const MANAGE_DROPDOWN_AUTH_METHOD =
   '[data-test-manage-auth-methods-dropdown] button';
-export const MANAGE_DROPDOWN_NEW_ACCOUNT = (url) =>
-  `[data-test-manage-auth-methods-dropdown] a[href="${url}"]`;
+export const MANAGE_DROPDOWN_ACCOUNT =
```

Comment:
nit: Wondering what you think about including `AUTH` in these as well, if the test selector includes `*-auth-*`? Similar to your var above.

Reply:
I opted not to include `AUTH` just to keep it shorter. Since this selector file is specific to accounts, I figured that would be a safe bet. Otherwise, we would have `MANAGE_DROPDOWN_AUTH_METHOD_ACCOUNT` which is very long. While the specificity in the `data-test` attribute is needed in my opinion, the name of the test selector doesn't need to be. We could potentially even rename that `data-test` attribute.

Reply:
Fair & non-blocking. I interpreted `auth` as being unique to these selectors, specifically. Either way ✅ !